pages/about/team/index.tsx

```diff
@@ -0,0 +1,98 @@
+import SEO from 'components/basecomponents/SEO';
+import { H2, LinkText, P } from 'components/Typography';
+import { GetStaticProps, NextPage } from 'next';
+import { Trans, useTranslation } from 'next-i18next';
+import Image from 'next/image';
+import React from 'react';
+import { getServerSideTranslations } from 'utils/localization';
+import { useLanguage } from 'utils/useLanguageHook';
+
+interface Members {
```

Comment:
`Team`, `TeamSections`, or `TeamStucture` would be a more intuitive name. `Members` implies just an array of names.